lms/templates/instructor/instructor_dashboard_2/data_download.html

```diff
@@ -32,6 +32,10 @@ <h3 class="hd hd-3">${_("Reports")}</h3>
 
     <p><input type="button" name="list-profiles-csv" value="${_("Download profile information as a CSV")}" data-endpoint="${ section_data['get_students_features_url'] }" data-csv="true"></p>
 
+    <p>${_("Click to generate a CSV file of all active students enrolled in this course, along with profile information such as email address and username:")}</p>
```

Comment:
Just my opinion, but this reads so similar to the string above that it may be hard to distinguish at first. We can maybe help that by making "all active students" bold or something. That may be something upstream UX/UI review could confirm, though.

Reply:
Yes, I could see this being a bit confusing. I'm not sure how, or if it's possible to add HTML into these translation strings however. Adding it directly to the string automatically HTML encodes it.
The only other option I can think of at the moment is splitting the string up, but that would mess up the translation layer.

Reply:
Ah, I think you're right, yeah :) Nevermind, just a nit. Could leave it to the upstream review.